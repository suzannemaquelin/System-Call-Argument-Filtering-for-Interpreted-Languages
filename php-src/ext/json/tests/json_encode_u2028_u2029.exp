string(12) "["a\u00e1b"]"
string(8) "["aáb"]"
string(10) ""a\u2027b""
string(7) ""a‧b""
string(10) ""a\u2028b""
string(10) ""a\u2028b""
string(10) ""a\u2028b""
string(7) ""a b""
string(10) ""a\u2029b""
string(10) ""a\u2029b""
string(10) ""a\u2029b""
string(7) ""a b""
string(10) ""a\u202ab""
string(7) ""a‪b""