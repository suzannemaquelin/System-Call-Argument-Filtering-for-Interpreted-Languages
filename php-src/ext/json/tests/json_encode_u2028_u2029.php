<?php
var_dump(json_encode(array("a\xC3\xA1b")));
var_dump(json_encode(array("a\xC3\xA1b"), JSON_UNESCAPED_UNICODE));
var_dump(json_encode("a\xE2\x80\xA7b"));
var_dump(json_encode("a\xE2\x80\xA7b", JSON_UNESCAPED_UNICODE));
var_dump(json_encode("a\xE2\x80\xA8b"));
var_dump(json_encode("a\xE2\x80\xA8b", JSON_UNESCAPED_UNICODE));
var_dump(json_encode("a\xE2\x80\xA8b", JSON_UNESCAPED_LINE_TERMINATORS));
var_dump(json_encode("a\xE2\x80\xA8b", JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_LINE_TERMINATORS));
var_dump(json_encode("a\xE2\x80\xA9b"));
var_dump(json_encode("a\xE2\x80\xA9b", JSON_UNESCAPED_UNICODE));
var_dump(json_encode("a\xE2\x80\xA9b", JSON_UNESCAPED_LINE_TERMINATORS));
var_dump(json_encode("a\xE2\x80\xA9b", JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_LINE_TERMINATORS));
var_dump(json_encode("a\xE2\x80\xAAb"));
var_dump(json_encode("a\xE2\x80\xAAb", JSON_UNESCAPED_UNICODE));
?>
