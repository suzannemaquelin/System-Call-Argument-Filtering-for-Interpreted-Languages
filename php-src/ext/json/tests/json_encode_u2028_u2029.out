Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
string(12) "["a\u00e1b"]"
string(8) "["aáb"]"
string(10) ""a\u2027b""
string(7) ""a‧b""
string(10) ""a\u2028b""
string(10) ""a\u2028b""
string(10) ""a\u2028b""
string(7) ""a b""
string(10) ""a\u2029b""
string(10) ""a\u2029b""
string(10) ""a\u2029b""
string(7) ""a b""
string(10) ""a\u202ab""
string(7) ""a‪b""